frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -336,7 +336,10 @@ export const surveyLogic = kea<surveyLogicType>([
             },
         },
         surveyUserStats: {
-            loadSurveyUserStats: async (): Promise<SurveyStats> => {
+            loadSurveyUserStats: async (): Promise<SurveyStats | null> => {
+                if (props.id === NEW_SURVEY.id || values.survey.start_date === null) {
+                    return null
+                }
```

Comment:
Is there any chance that `values.survey` could be null here?

Reply:
it's all good on this side, since the default value for survey is [defined here](https://github.com/PostHog/posthog/blob/fix/dont-call-survey-stats-if-survey-is-new-or-not-launched/frontend/src/scenes/surveys/surveyLogic.tsx#L832), which is the const we use for new surveys